airflow_account/models/sale_order.py

```diff
@@ -8,8 +8,12 @@ class SaleOrder(models.Model):
     number_of_invoices = fields.Integer('# of Invoices', readonly=True, help='For internal usage only')
     number_of_credit_notes = fields.Integer('# of Credit Notes', readonly=True, help='For internal usage only')
 
-    proj_manager = fields.Many2one('res.partner', string="Project Manager", default=False, store=True)
-    proj_name = fields.Char(string="Project Name", default=False, store=True)
+    # The field project_manager is depricated. Airflow wishes to have it hold Char type instead of many2one.
+    # Keeping field for historical reasons and data preservation.
+    proj_manager = fields.Many2one('res.partner', string="Project Manager(Depricated)", store=True)
 
-    ship_date = fields.Date(string="Ship Date", default=False, store=True)
-    ship_method = fields.Many2one('delivery.carrier', string="Shipping Method", default=False, store=True)
+    proj_manager_char = fields.Char(string="Project Manager", store=True)
```

Comment:
```suggestion
    proj_manager_name = fields.Char(string="Project Manager")
```
Regular field do  need store explicit and field name impovments